src/components/containers/partitioned_vector/partitioned_vector_component_double.cpp

```diff
@@ -21,7 +21,10 @@ HPX_REGISTER_PARTITIONED_VECTOR(double);
 #pragma warning(disable: 5037)
 #endif
 
-template class HPX_PARTITIONED_VECTOR_EXPORT
+template class
+#if defined(HPX_MSVC) || defined(HPX_MINGW)
+HPX_PARTITIONED_VECTOR_EXPORT
+#endif
```

Comment:
Could we handle that by defining a separate pp constant (similarly to `HPX_PARTITIONED_VECTOR_SPECIALIZATION_EXPORT`, see here: https://github.com/STEllAR-GROUP/hpx/blob/master/hpx/components/containers/partitioned_vector/export_definitions.hpp#L17-L21)?

Reply:
Ooh, yes, absolutely. Thanks for pointing that out.